indico/modules/events/abstracts/templates/management/abstract_person_list.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'events/management/_event_person.html' %}
+
+{% block selection_toolbar %}
+    <input type="checkbox" id="filter-submitters" data-filter="submitter" checked>
+    <label for="filter-submitters" class="i-button">{% trans %}Submitters{% endtrans %}</label>
+    {{ super() }}
+{% endblock %}
+
+{% block submitters_roles %}
+    {% set is_submitter = person_roles.submitter %}
+    {%- if is_submitter %}
+        {% set submitter_title = _("This person submitted an abstract") %}
+    {%- else -%}
+        {% set submitter_title = _("This person hasn't submitted any abstracts") %}
+    {% endif -%}
+    <i class="icon-user-check submitter-icon {{ 'inactive' if not is_submitter }}"
+       title="{{ submitter_title }}"></i>
+{% endblock %}
+
+{% block no_persons_message %}
+    <td colspan="6">{% trans %}No persons in the selected abstracts yet{% endtrans %}</td>
```

Comment:
I think this case is technically impossible since there's always a submitter, so you could just leave it empty (maybe with a comment mentioning that this is never empty).